openrtb_ext/request_wrapper.go

```diff
@@ -780,7 +832,7 @@ func (re *RegExt) SetUSPrivacy(uSPrivacy string) {
 type SiteExt struct {
```

Comment:
Not really related to your changes but wondering if there's any difference between `SiteExt` that's defined here vs `ExtSite` defined in `openrtb_ext/site.go`? The latter doesn't really seem to be used anywhere.. perhaps we can remove it as part of a separate PR?